pkg/registry/apis/secret/accesscontrol.go

```diff
@@ -0,0 +1,116 @@
+package secret
+
+import (
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+)
+
+const (
+	// SecureValues
+	ActionSecretsManagerSecureValuesCreate = "secrets-manager.securevalues:create" // CREATE.
+	ActionSecretsManagerSecureValuesWrite  = "secrets-manager.securevalues:write"  // CREATE + UPDATE.
+	ActionSecretsManagerSecureValuesRead   = "secrets-manager.securevalues:read"   // GET + LIST.
+	ActionSecretsManagerSecureValuesDelete = "secrets-manager.securevalues:delete" // DELETE.
+
+	// Keepers
+	ActionSecretsManagerKeepersCreate = "secrets-manager.keepers:create" // CREATE.
+	ActionSecretsManagerKeepersWrite  = "secrets-manager.keepers:write"  // UPDATE.
+	ActionSecretsManagerKeepersRead   = "secrets-manager.keepers:read"   // GET + LIST.
+	ActionSecretsManagerKeepersDelete = "secrets-manager.keepers:delete" // DELETE.
+)
+
+var (
+	ScopeProviderSecretsManagerSecureValues = accesscontrol.NewScopeProvider("secrets-manager.securevalues")
+	ScopeProviderSecretsManagerKeepers      = accesscontrol.NewScopeProvider("secrets-manager.keepers")
+
+	ScopeAllSecureValues = ScopeProviderSecretsManagerSecureValues.GetResourceAllScope()
+	ScopeAllKeepers      = ScopeProviderSecretsManagerKeepers.GetResourceAllScope()
+)
+
+func RegisterAccessControlRoles(service accesscontrol.Service) error {
```

Comment:
@kalleep @gamab I guess I still need to create the roles and grant these permissions here right? Or is there another nice helper functions for this? 😄

Reply:
You still do :+1: